### PR TITLE
Layout revision of page "My artitst"

### PR DESCRIPTION
**Pull request by DanielW1987**
The page "My Artists" has now been completely revised (design and TypeScript). The pagination will be available via a separate PR, because we need a separate TypeScript module for the pagination. 

I think it makes sense that you do the review after we have had our know how session.

### CODE REVIEW

**Comment by DanielW1987** [comment]
[[TypeScript] Pagination Module](https://trello.com/c/ZPOE6nE2/192-typescript-pagination-module)

File: webapp/src/main/resources/static/ts/src/service/my-artists-render-service.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                const infoMessage = this.alertService.renderInfoAlert(message, false);
+                this.hostElement.insertAdjacentElement("afterbegin", infoMessage);
+            }
+            else if (response.pagination.totalPages < response.pagination.currentPage) {
+                // If you follow for example 21 bands and you unfollow one band on page 1 and then go to page 2 there are no bands to display.
+                // In this case you will be forwarded to the last page.
+                window.location.replace(`?page=${response.pagination.totalPages}`);
+            }
+            else {
+                let currentCarNo = 1;
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/service/my-artists-render-service.ts:45]:
```suggestion
                let currentCardNo = 1;
```

File: webapp/src/test/java/rocks/metaldetector/service/artist/FollowArtistServiceImplTest.java [hunk 2]
```diff
@@ -39,7 +40,7 @@
 class FollowArtistServiceImplTest implements WithAssertions {
 
   private static final String EXTERNAL_ID = "252211";
-  private static final String ARTIST_SOURCE = "Discogs";
+  private static final ArtistSource ARTIST_SOURCE = ArtistSource.DISCOGS;
```
Comment (hedgehog1833) [inline webapp/src/test/java/rocks/metaldetector/service/artist/FollowArtistServiceImplTest.java:43]: Static import pl0x

File: webapp/src/main/resources/static/ts/src/service/my-artists-render-service.ts (hunk 1 above)
Comment (hedgehog1833) [reply webapp/src/main/resources/static/ts/src/service/my-artists-render-service.ts:45]: And yes, there are more places to adjust than that 😉

**Pull request merged**